### ISSUE TITLE
Google security warning on monkeytype.com

### ISSUE THREAD

**Issue by duckyb** [Miodec/monkeytype#427]
The site `https://monkey-type.com/` redirects to `monkeytype.com` and Google prompts me with a warning:

![image](https://user-images.githubusercontent.com/27895007/96305812-c66bde80-0ffe-11eb-8196-cf4200dd5c19.png)

I'm not concerned about my password but I'm wondering why this happens.

**Comment by Miodec** [comment]
The new domain seems to be triggering the Chrome's built in phishing detection. 

Basically, since you used monkey-type a lot, when chrome sees that you've entered the same credentials to a veeeery similar domain, Chrome thinks that monkeytype.com is trying to steal your information for... monkey-type.com, and labels the website as Dangerous. (even though its pointing to the same server..)

I've tested with family members and friends who don't visit the site as often or at all, and the dangerous popup was not there. Also, Google's Safe Browsing scanner classifies monkeytype.com as safe, so this leads me to believe the 'dangerous' classification is happening locally.

I've contacted Google to get this sorted out - don't worry, the site is safe.
